generation/localizedstringkit/detection.py

```diff
@@ -128,16 +138,31 @@ def _detect_strings(self, patterns: List[Tuple[Pattern, int]]) -> List[Localized
             )
 
             if len(match) == 2:
```

Comment:
Leaving the length checks for the current ones to avoid moving the PATTERNs to global space for comparison